routers/user.py

```diff
@@ -0,0 +1,259 @@
+import httpx
+
+from auth.config import get_settings
+from auth.management import get_management_token
+from auth.validator import get_current_user
+from datetime import datetime, timezone
+from fastapi import APIRouter, Depends, HTTPException
+from schemas.requests import ResourceRequest, ServiceRequest
+from schemas.user import User
+
+router = APIRouter()
+
+def get_settings_instance():
```

Comment:
What is this achieving? Can't you just call `get_settings()` directly in each route that needs it (it's cached so shouldn't have too much overhead on repeated calls)